src/t_list.c

```diff
@@ -385,7 +385,53 @@ void lsetCommand(client *c) {
     }
 }
 
-void listElementsRemoved(client *c, robj *key, int where, robj *o) {
+/* A helper for replying with a list's range as a multi-bulk. For backward
+* compatibility, start needs to be less than end. The direction affects how
+* elements are iterated. */
```

Comment:
what do you mean by backward compatibility?
you mean the old behavior of LRANGE? let's be specific since this code is not inside the command function.
alternatively, maybe this check should be in the command function and not here.

Reply:
Made the comment clearer in bae0cb8